tests/testutils/httputils.go

```diff
@@ -44,6 +43,8 @@ func HttpServe(addr string, timeout int) error {
 
 	c := make(chan string)
 	go func() {
+		log.Printf("%v: serving on %v\n", hostname, addr)
```

Comment:
I see your point, we'd need an interface for Logger and could then use whatever is passed into the utility functions. We could probably use this interface in several projects, not just here.